source/bson-decimal128/decimal128.rst

```diff
@@ -76,7 +76,9 @@ Clamping:
 
 Binary Integer Decimal (BID):
    MongoDB uses this binary encoding for the coefficient as specified in ``IEEE
-   754-2008``. The byte order is little-endian, like the rest of the BSON types.
+   754-2008`` section 3.5.2 using method 2 "binary encoding" rather than method 1
```

Comment:
For any who do not have access to the `IEEE Std. 754-2008` document, a quote of the key section:

from 3.5.2 on page 11:

> c. For finite numbers, r is (S, E − bias, C) and v = (−1) S × 10 (E−bias) × C, where C is the concatenation of the leading significand digit or bits from the combination field G and the trailing significand field T, and where the biased exponent E is encoded in the combination field. The encoding within these fields depends on whether the implementation uses the decimal or the binary encoding for the significand.

> > 1) If the implementation uses the decimal encoding for the significand, then the least significant w bits of the exponent are G 5 through G w +4 . The most significant two bits of the biased exponent and the decimal digit string d 0 d 1 ...d p −1 of the significand are formed from bits G 0 through G 4 and T as follows:

> > > i) When the most significant five bits of G are 110xx or 1110x, the leading significand digit d 0 is 8 + G 4 , a value 8 or 9, and the leading biased exponent bits are 2G 2 + G 3 , a value 0, 1, or 2.

> > > ii) When the most significant five bits of G are 0xxxx or 10xxx, the leading significand digit d 0 is 4G 2 + 2G 3 + G 4 , a value in the range 0 through 7, and the leading biased exponent bits are 2G 0 + G 1 , a value 0, 1, or 2. Consequently if T is 0 and the most significant five bits of G are 00000, 01000, or 10000, then v = (−1) S × (+0). 

> > The p −1 = 3 × J decimal digits d 1 ...d p −1 are encoded by T, which contains J declets encoded in densely-packed decimal.
> > A canonical significand has only canonical declets, as shown in Tables 3.3 and 3.4. Computational operations produce only the 1000 canonical declets, but also accept the 24 non-canonical declets in operands.

> > 2) Alternatively, if the implementation uses the binary encoding for the significand, then:

> > > i) If G 0 and G 1 together are one of 00, 01, or 10, then the biased exponent E is formed from G 0 through G w +1 and the significand is formed from bits G w +2 through the end of the encoding (including T).

> > > ii) If G 0 and G 1 together are 11 and G 2 and G 3 together are one of 00, 01, or 10, then the biased exponent E is formed from G 2 through G w +3 and the significand is formed by prefixing the 4 bits (8 + G w +4 ) to T.

> > The maximum value of the binary-encoded significand is the same as that of the corresponding decimal-encoded significand; that is, 10 (3 × J + 1) −1 (or 10 (3 × J ) −1 when T is used as the payload of a NaN). If the value exceeds the maximum, the significand c is non-canonical and the value used for c is zero.

> Computational operations generally produce only canonical significands, and always accept non-canonical significands in operands.